Header header
string object_id
uint32 cur_x
uint32 cur_y
uint32 prev_x
uint32 prev_y
